interval-functor.cabal

```diff
@@ -50,6 +50,8 @@ library
   build-depends:
     , base >= 4.10 && < 5
     , transformers ^>= 0.5.6.2
+    , generic-data ^>= 0.9
```

Comment:
I’d rather we define the instance ourselves than take on this dependency.

Reply:
Alright!